docs/src/components/code-block/package-map.ts

```diff
@@ -65,6 +69,7 @@ export const PACKAGE_MAP: PackageMap = {
   '@pluralsight/ps-design-system-banner': { Banner },
   '@pluralsight/ps-design-system-breadcrumb': { Breadcrumb },
   '@pluralsight/ps-design-system-button': { Button },
+  '@pluralsight/ps-design-system-core': { core },
```

Comment:
I think we want to encourage people to use the named imports so they get tree shaking.   As in this circ prog PR: https://github.com/pluralsight/design-system/blob/4dfceab1f5af46fb3bbb9578d1f114661f415795/docs/src/components/code-block/package-map.ts#L73

Reply:
I'll see if I can make that work. It worked great for Icon but got an error when using it on core.